### PR TITLE
Commit new social media guidelines

### PR DESCRIPTION
**Pull request by joseluisbolos**
Includes links to [http://cruft.io](http://cruft.io) and [@cruftio on Twitter](https://twitter.com/cruftio).

### CODE REVIEW

File: practices/social-media.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Social media
+
+* [Cruft.io](#cruft-io)
+* [Posting publicly](#posting-publicly)
+
+An important part of our work as developers is not only to create quality products, but also to raise awareness of accessibility, performance and progressive enhancenment best practices. This includes promoting our work and our practices in open spaces like social media, conferences, talks and other events.
```
Comment (danyalaytekin) [inline practices/social-media.md]: enhancement

File: practices/social-media.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+For open-source projects, we aim to keep our projects up to date and fix any vulnerabilities in a timely fashion. Please see our [Open Source Support page](https://github.com/springernature/frontend-playbook/blob/master/practices/open-source-support.md) for details on how we do this.
+
+
+## Cruft.io
+
+[Cruft.io](http://cruft.io) is a blog run by a group of developers, testers, sysadmins, scrum masters and product owners at Springer Nature. Anyone can post there and the intructions can be found in the [project's wiki (internal)](https://github.com/springernature/cruft/).
```
Comment (danyalaytekin) [inline practices/social-media.md]: instructions

File: practices/social-media.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Cruft has also a [@cruftio Twitter account](https://twitter.com/cruftio) that is used to promote its blog posts. We're also using @cruftio to promote the release of new versions of our open-source projects.
+
+
+## Posting publicly
+
+Before publishing anything be sure that:
```
Comment (danyalaytekin) [inline practices/social-media.md]: 'ensure that'?

File: practices/social-media.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+* All the public communications follow our [language standards](language.md).
+* When reporting bugs or security issues, remember to use positive language instead of blaming:
```
Comment (danyalaytekin) [inline practices/social-media.md]: Remove `:`?

File: practices/social-media.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+We do:
+> Released Shunter 1.2.3. Security upgrade of `nac` to 3.2.1 which addresses several vulns. https://github.com/springernature/shunter/blob/master/HISTORY.md#123-2017-03-08
+
+We _don't_ do:
+> Released Shunter 1.2.3. Fixes a vulnerability which was caused by a bug in `nac`, breaking our project. https://github.com/springernature/shunter/blob/master/HISTORY.md#123-2017-03-08
+
+* Always provide links where people can kind further information if interested. E.g. A changelog for a software release, a blog article, a link to the Frontend Playbook, etc.
```
Comment (danyalaytekin) [inline practices/social-media.md]: find, front-end

**Comment by joseluisbolos** [comment]
Typos fixed! 👍

**Pull request merged**